Documentation/Getting-Started/Prerequisites/prerequisites.md

```diff
@@ -7,7 +7,7 @@ and Rook is granted the required privileges (see below for more information).
 
 ## Minimum Version
```

Comment:
The helm docs also need to be updated, looks like the previous version they weren't updated. Please search for `Kubernetes 1.19+` and update those as well.

Reply:
Sure Done